### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by oliveiraigorm**

### CODE REVIEW

**Comment by mwalbeck** [comment]
A few extra things that didn't really fit in the review (coming soon)

image

I would like to have a margin added between the entries here of `0.25rem`

___

image

Is it possible to make this look more like the Figma?

___

image

I think we should make the timezone select look like the other input fields. It's probably going to take a bit of work to get the height correct, but I think it's going to create a more cohesive visual style.

File: src/components/SessionCard.vue [hunk 1]
```diff
@@ -89,10 +98,6 @@ export default {
 </script>
 
 <style scoped>
-.blue {
-	color: var(--adminly-dark-blue);
-}
-
 .session-header {
 	margin-top: 1.25rem;
```
Comment (mwalbeck) [inline src/components/SessionCard.vue:102]: This should be changed to `margin: 0.5rem 0 0.25rem;`

**Review by mwalbeck — CHANGES_REQUESTED** [review]

File: src/App.vue [hunk 2]
```diff
@@ -59,28 +47,53 @@ main {
...
+	display: flex;
+	flex-direction: row;
+}
+
+.modal-footer button {
+	font-family: "Roc Grotesk", var(--font-face);
+}
+
+.modal-footer {
+	display: flex;
```
[the remark this answers is not in the document]
Comment (oliveiraigorm) [reply src/App.vue:65]: The display flex is still important for the create client modal

[the remark this answers is not in the document]
Comment (mwalbeck) [reply src/App.vue:65]: Oh sorry, I though I had removed this comment. Yeah, I did notice it was need later one, so you can just ignore it.

File: src/components/ClientCreation.vue [hunk 3]
```diff
@@ -98,27 +130,20 @@ export default {
...
+	border-radius: 4px;
 }
 
-button {
-	box-shadow: 0px 0px 11px var(--adminly-grey);
-	border-radius: 8px;
-	background-color: white;
+input:focus,
+textarea:focus {
+	border-color: var(--adminly-blue) !important;
```
[the remark this answers is not in the document]
Comment (mwalbeck) [reply src/components/ClientCreation.vue:141]: Yes, definitely! You're welcome to add it to the branch I linked

File: src/components/Table.vue [hunk 4]
```diff
@@ -146,18 +146,24 @@ table {
...
 	border-style: hidden; /* hide standard table (collapsed) border */
 	background-color: white;
-	color: var(--neutral-600);
+	color: var(--adminly-dark-blue);
 }
 
 table th {
-	font-weight: bold;
 	padding: 0.7rem 1.25rem;
 	border: 1 solid var(--adminly-line-color);
```
[the remark this answers is not in the document]
Comment (oliveiraigorm) [reply src/components/Table.vue:154]: not necessary since the tr also has the same rule

File: src/components/ClientsTable.vue [hunk 5]
```diff
@@ -229,8 +236,9 @@ export default {
 	margin: 0.7rem;
 }
 
-input {
-	border: 0.5px solid var(--adminly-dark-blue);
+input,
+select {
+	border: 0.5px solid var(--color-border-dark);
```
Comment (mwalbeck) [inline src/components/ClientsTable.vue:241]: border width should be 1px.

File: src/components/ClientCreation.vue [hunk 6]
```diff
@@ -96,56 +128,75 @@ export default {
...
 </script>
-
-<style scoped>
-.modal-container {
-	width: 50vw;
-}
-
-input {
+<style>
+#create-modal .modal-container {
```
Comment (mwalbeck) [inline src/components/ClientCreation.vue:132]: We also need to add the following to this rule:

```
display: flex;
flex-direction: column;
```

File: src/components/ClientCreation.vue [hunk 7]
```diff
@@ -96,56 +128,75 @@ export default {
...
+	max-width: 300px;
```
Comment (mwalbeck) [inline src/components/ClientCreation.vue:133]: We should increase the max-width to 320px

File: src/components/ClientCreation.vue [hunk 8]
```diff
@@ -96,56 +128,75 @@ export default {
...
-	margin-left: auto;
+	padding: 0.25rem 1rem 0 0;
 	display: flex;
 	justify-content: end;
 }
 
-.modal-header button {
-	border: none;
-	box-shadow: none;
+.modal-content {
```
Comment (mwalbeck) [inline src/components/ClientCreation.vue:155]: We should add `align-self: center;` to this rule.

File: src/components/ClientCreation.vue [hunk 9]
```diff
@@ -96,56 +128,75 @@ export default {
...
+	max-width: 300px;
 	width: 100%;
 }
-
-button {
-	box-shadow: 0px 0px 11px var(--adminly-grey);
-	border-radius: 8px;
-	background-color: white;
+</style>
+<style scoped>
```
Comment (mwalbeck) [inline src/components/ClientCreation.vue:137]: We need one more scoped css rule for the client creation modal footer:

```
.modal-footer {
    margin-top: 2rem;
}
```

File: src/components/ClientModal.vue [hunk 10]
```diff
@@ -240,38 +281,59 @@ export default {
...
 	max-width: 200px;
+	border: 1px solid var(--color-border-dark) !important;
+}
+
+.client-info .col .multiselect--single {
+	height: 34px;
+}
+
+#client-modal .modal-container {
+	max-height: 80% !important;
```
Comment (mwalbeck) [inline src/components/ClientModal.vue:312]: After a bit more testing I think 80% is a bit too small, so we should set it to 90% instead.

File: src/components/ClientsTable.vue [hunk 11]
```diff
@@ -197,9 +215,14 @@ export default {
...
 	align-items: center;
-	color: var(--neutral-600);
+	justify-content: center;
+}
+
+.page-selector {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
 }
```
Comment (mwalbeck) [inline src/components/ClientsTable.vue:227]: Can we get the following css rule added here:

```
.page-selector select {
    margin-right: 1rem;
}
```

**Review by mwalbeck — CHANGES_REQUESTED** [review]
A few extra things,

Also when clicking on the timezone selector to select a timezone, it changes size. We should have it be a fixed size, so it doesn't affect the layout of the modal.

**Comment by oliveiraigorm** [comment]
@mwalbeck About the timezone picker border, I've tried a few things on Friday to change it's height but with no luck. Could you give me some help on it?

**Comment by mwalbeck** [comment]
> @mwalbeck About the timezone picker border, I've tried a few things on Friday to change it's height but with no luck. Could you give me some help on it?

I'm not quite sure I catch what you're referring to?

___

Though I do have 2 quick things I noticed on the timzone picker. The whole multiselect needs a margin-top of 3px to match the spacing of the other input fields:

image

Also the `.multiselect__input` itself has a margin-top of 3px that we want to be 0, since it causes the "Type to search timezone" text to not be centered.

**Comment by oliveiraigorm** [comment]
> I'm not quite sure I catch what you're referring to?

Sorry, I meant making the timezone picker height the same as the other inputs.

**Comment by mwalbeck** [comment]
But they seemed to be the same height when I checked just before?

**Comment by oliveiraigorm** [comment]
> But they seemed to be the same height when I checked just before?


It has 36px vs 34px on the normal inputs

image
image

**Review by mwalbeck — CHANGES_REQUESTED** [review]
The height of the timezone picker is one of those issues that are quite hard to debug if you don't already know the details of how box-sizing works in CSS. But I've suggested some changes that should touch up the last few things with the selector.

**Comment by oliveiraigorm** [comment]
Thanks @mwalbeck !

**Review by mwalbeck — APPROVED** [review]
Good job!

**Pull request merged**